packages/core/core/src/config.ts

```diff
@@ -374,17 +374,17 @@ export class Config {
     }
 
     if (this.blockchainConfig.targetGasLimit && this.blockchainConfig.targetGasLimit.toString().match(unitRegex)) {
-      this.blockchainConfig.targetGasLimit = getWeiBalanceFromString(this.blockchainConfig.targetGasLimit, web3);
+      this.blockchainConfig.targetGasLimit = getWeiBalanceFromString(this.blockchainConfig.targetGasLimit);
     }
 
     if (this.blockchainConfig.gasPrice && this.blockchainConfig.gasPrice.toString().match(unitRegex)) {
-      this.blockchainConfig.gasPrice = getWeiBalanceFromString(this.blockchainConfig.gasPrice, web3);
+      this.blockchainConfig.gasPrice = getWeiBalanceFromString(this.blockchainConfig.gasPrice);
     }
 
     if (this.blockchainConfig.accounts) {
       this.blockchainConfig.accounts.forEach(acc => {
         if (acc.balance && acc.balance.toString().match(unitRegex)) {
-          acc.balance = getWeiBalanceFromString(acc.balance, web3);
+          acc.balance = getWeiBalanceFromString(acc.balance3);
```

Comment:
Is `acct.balance3` correct?

Reply:
Nope, definitely an accident when backing out `, web3`. Great catch!